FatmaQunnies/Task.py

```diff
@@ -0,0 +1,40 @@
+from datetime import date
+
+class Task:
+ 
+    counter = 0  
+    def __init__(self, title, description, due_date, status = 'InProgress'):
```

Comment:
I like the idea of using a default value for the status. However, I wanted to mention something I didn't bring up before: you can use [enums](https://docs.python.org/3/library/enum.html) in Python. This way, if we ever want to update the InProgress value to something like in-progress, we only need to update the enum. But anyway, great work!